### PR TITLE
Automate negative tcs to set qos for namespace

### PR DESCRIPTION
**Pull request by HaruChebrolu**
# Description

Automate negative testcases to set qos for namespace
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581753
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581945

logs:
http://magna002.ceph.redhat.com/ceph-qe-logs/Harika/cephci-run-8FJI5W/

<summary>click to expand checklist</summary>

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [x] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+#  Test cases to be executed
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node10
+          - node11
```
Comment (rahullepakshi) [inline suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml:58]: @HaruChebrolu seems like conf file conf/reef/nvmeof/ceph_nvmeof_sanity.yaml does not have node11 by id or actual node11. So this test will fail, right?

File: suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client for NVMe tests
+      polarion-id: CEPH-83573758
+
+  - test:
+      abort-on-fail: false
+      config:
+        gw_node: node6
```
Comment (rahullepakshi) [inline suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml:71]: Can we also enhance this to 2 GW configuration going forward?

File: suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml (hunk 1 above)
Comment (HaruChebrolu) [reply suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml:58]: @rahullepakshi you are right. I forgot to change the conf file used in comments section. Will update it.

Comment (HaruChebrolu) [reply suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml:58]: Done

File: suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml (hunk 2 above)
Comment (HaruChebrolu) [reply suites/reef/nvmeof/tier-4_nvmeof_qos_negativetests.yaml:71]: I think, we can. But we might require few changes accordingly to accomodate in test modules

**Pull request closed without merging**